make-checkout

```diff
@@ -37,6 +37,7 @@ def main():
     parser = argparse.ArgumentParser(description="Fetch and checkout specific revision")
     parser.add_argument("-v", "--verbose", action="store_true", help="Verbose output")
     parser.add_argument("--repo", nargs='?', help="Repository to check out")
+    parser.add_argument("--rebase", dest="branch", help="Branch to rebase to")
```

Comment:
To avoid confusion, please drop dest and leave it as `opts.rebase`.

Reply:
I just wanted to have in help `--rebase BRANCH` and not `--rebase REBASE`. Is it possible to specify this without dest?

Reply:
Yes, use `metavar=`